From 10a4d2069f228f89c51a86f60f7b2fb9fb5222ed Mon Sep 17 00:00:00 2001
From: Michael Bestas <mkbestas@lineageos.org>
Date: Thu, 21 Feb 2019 21:31:02 +0200
Subject: [PATCH 5/9] legacy: Resolve rome BT denials

* Devices using rome BT chip need to access
  /data/misc/bluedroid/bt_fw_version.txt

Change-Id: I48f6c90b763b550a46bea616691226ad5f5885c8
---
 legacy-common/hal_bluetooth_default.te | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/legacy-common/hal_bluetooth_default.te b/legacy-common/hal_bluetooth_default.te
index aef3417..cd5c85d 100644
--- a/legacy-common/hal_bluetooth_default.te
+++ b/legacy-common/hal_bluetooth_default.te
@@ -1 +1,4 @@
+allow hal_bluetooth_default bluetooth_data_file:dir ra_dir_perms;
+allow hal_bluetooth_default bluetooth_data_file:file create_file_perms;
+
 allow hal_bluetooth_default wcnss_filter:unix_stream_socket connectto;
-- 
2.17.1

